vignettes/developers.Rmd

```diff
@@ -0,0 +1,358 @@
+---
+title: "Translation for package developers"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Translation for package developers}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+library(glue)
+```
+
+In this vignette you'll learn how to set up your package for translation.
+It's aimed at a package developers; if you're translating an existing package, you might want to start with `vignette("translators")`.
+
+```{r setup}
+library(potools)
+```
+
+## Basic process
+
+Before we get into the details lets review the basic process:
+
+-   You run `po_extract()` to extract all translatable messages from your R and C code.
+    This creates a `.pot` (po template) file that contains every translatable message.
```

Comment:
some how just learned .po means Portable Object:

https://www.gnu.org/software/gettext/manual/gettext.html#PO

(had been thinking it's just short for `po`rtable)